src/site/content/en/blog/prefers-color-scheme/index.md

```diff
@@ -0,0 +1,174 @@
+---
+title: Hello Darkness, My Old Friend—Dark Theme is Here
+subhead:
```

Comment:
Need a catchy subhead. This will appear on the post's card on the blog landing page and under the title on the article itself.